gwpy/timeseries/io/losc.py

```diff
@@ -131,7 +132,14 @@ def _overlapping(files):
 
 # -- remote data access (the main event) --------------------------------------
 
-def fetch_gwosc_data(detector, start, end, cls=TimeSeries, **kwargs):
+def fetch_gwosc_data(
+        detector,
+        start,
+        end,
+        nthreads=10,
```

Comment:
Can you please add this option to the 'upstream' `TimeSeriesBase.fetch_open_data` method, or at least document the keyword argument there.

And also, probably somewhere in the docs.